### ISSUE TITLE
Compilation issue with Xcode 8.3  : Type of expression is ambiguous without more context

### ISSUE THREAD

**Issue by phimage** [146BC/StyleKit#43]
There is some issues with last Xcode
```
.../StyleKit/StyleKit/Stylist.swift:104:98: Type of expression is ambiguous without more context
.../StyleKit/StyleKit/Stylist.swift:109:57: Type of expression is ambiguous without more context
.../StyleKit/StyleKit/Stylist.swift:114:116: Type of expression is ambiguous without more context
.../StyleKit/StyleKit/Stylist.swift:123:75: Type of expression is ambiguous without more context
.../StyleKit/StyleKit/Stylist.swift:130:98: Type of expression is ambiguous without more context
.../StyleKit/StyleKit/Stylist.swift:135:57: Type of expression is ambiguous without more context
```

For instance
```swift
self.currentComponent?.appearance(whenContainedInInstancesOf: viewStack).perform(Selector(modifiedSelector))
```

**Comment by BernardGatt** [comment]
I'm currently looking into this, thanks for reporting.

**Comment by BernardGatt** [comment]
Just submitted a PR if you wish to have a look at the fix.

**Comment by BernardGatt** [comment]
Fix merged to develop, scheduling for next release.

**Comment by phimage** [comment]
Just tested on my current project, it's look goods
thanks

897a19fabb13edf86aa3fe4d1ca6ef7843391760

**Comment by BernardGatt** [comment]
Just released [0.6.0](https://github.com/146BC/StyleKit/releases/tag/0.6.0) closing this issue.
